app/mailers/order_mailer.rb

```diff
@@ -1,24 +1,25 @@
 class OrderMailer < ApplicationMailer
 
-  # Subject can be set in your I18n file at config/locales/en.yml
-  # with the following lookup:
-  #
-  #   en.order_mailer.received.subject
-  #
   default from: 'ankitadixit.rails@gmail.com'
   def received(order)
     @order = order
-    logger.info(order)
-    mail to: order.email, subject: 'Pragmatic Store Order Confirmation'
+    @line_items = order.line_items.includes(product: :images)
+
+    @line_items.each do |li|
+      attachments.inline[li.product.image_url] = File.read(Rails.root.join('app/assets/images', li.product.image_url))
```

Comment:
only first image should be inline and rest should be normal attachments.

`images.first` should be inline